src/test/resources/htsjdk/samtools/cram/7ddd8a4b4f2c1dec43476a738b1a9b72

```diff
@@ -0,0 +1 @@
+GCTAGCTCAGAAAAAAAAAA
```

Comment:
We might want to put this file into a directory with a name that makes it clear that its intended to be a samtools cache.

Reply:
I have moved the file into src/test/resources/htsjdk/samtools/ref_cache.